WebContent/js/containers/Filters.js

```diff
@@ -54,6 +56,7 @@ export class Filters extends React.Component {
     }
 
     componentDidMount() {
+        setTimeout(this.setFocusOnOwner, 100);
```

Comment:
Could you explain why this needs a timeout? On the surface this looks like it could trigger a race condition or something?

Reply:
removed for now